### PR TITLE
Notify user when link is copied under Call for speakers And Share Event

### LINKED ISSUE #2108: Notify user when link is copied under Call for speakers and Share event
**Describe the bug**

Currently as the link is copied the user is not notified.
**To Reproduce**
1. Go to 'Create event'
2. Click on 'Sessions and Speakers'
3. Scroll down to 'Call for speakers'

**Expected behavior**

Notify user when link is copied under Call for speakers and share event.
**Screenshots**

screenshot 2019-02-04 at 11 31 38 pm

**Additional context**

I'm working on it.

### PR DESCRIPTION
**Pull request by harsh-jindal**
#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [X] My branch is up-to-date with the Upstream `development` branch.
- [X] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Notify user when link is copied under Call for speakers

#### Changes proposed in this pull request:

screenshot 2019-02-06 at 11 17 58 am

Fixes #2108

### CODE REVIEW

File: app/templates/components/forms/wizard/sessions-speakers-step.hbs [hunk 1]
```diff
@@ -158,11 +158,16 @@
...
-          class='ui black right labeled icon copy button'}}
+          class='ui black right labeled icon copy button'
+          click=(action 'copiedText')
+        }}
           <i class="copy icon"></i>
           {{t 'Copy'}}
         {{/copy-button}}
       </div>
+      {{#if this.isLinkSuccess}}
+        <div><span style="float:right;color:green;">Link copied to clipboard</span></div>
```
Comment (CosmicCoder96) [inline app/templates/components/forms/wizard/sessions-speakers-step.hbs]: use popup

**Comment by harsh-jindal** [comment]
@CosmicCoder96 Made the changes

**Comment by harsh-jindal** [comment]
@uds5501 In #2086 I have to make the whole div clickable. And @niranjan94 has requested changes regarding that. So keeping them separate would be best for now.

**Comment by ritikamotwani** [comment]
@harsh-jindal any updates?

**Comment by harsh-jindal** [comment]
@ritikamotwani @CosmicCoder96 I tried to add the delay property like this:
{{#ui-popup on='hover' content=(t 'Link copied to clipboard') position='bottom right' delay = '{ show: 50,hide: 1000}' }}
          {{#copy-button
            clipboardText=(concat data.event.url '/cfs/' data.speakersCall.hash)
            class='ui black right labeled icon copy button'}}
            <i class="copy icon"></i>
            {{t 'Copy'}}
          {{/copy-button}}
{{/ui-popup}}
It doesn't seem to work.
Can you help me on how to add it?

**Comment by niranjan94** [comment]
@harsh-jindal `delay` requires an object. Whereas you seem to be passing in a string.

```hbs
{{#ui-popup on='hover' delay = '{ show: 50,hide: 1000}' }}
{{/ui-popup}}
```

--

Instead try passing in an object like 

```hbs
{{#ui-popup on='hover' delay=(hash show=50 hide=1000)}}
{{/ui-popup}}
```

**Comment by harsh-jindal** [comment]
@niranjan94 I tried it, didn't work.

**Comment by harsh-jindal** [comment]
@CosmicCoder96 @niranjan94 delay property is only for 'hover' and 'focus'. So i'm unable to close popup when using 'click'.

**Comment by kushthedude** [comment]
@harsh-jindal Any updates or should i take the issue?

**Comment by harsh-jindal** [comment]
@kushthedude The issue is to add delay property for popup on focus which is not possible in semantic ui. Other than that issue is fixed.

**Comment by harsh-jindal** [comment]
@CosmicCoder96 Please review this PR.

**Pull request merged**